app/subscriber/src/features/landing/Landing.tsx

```diff
@@ -32,6 +32,7 @@ export const Landing: React.FC = () => {
   const [activeItem, setActiveItem] = React.useState<string>(SidebarMenuItems.home.label);
   const [{ userInfo }] = useApp();
   const navigate = useNavigate();
+  const contentFrame = React.useRef();
```

Comment:
You will need to attach the ref to the component being scrolled to I believe

Reply:
It was sent by mistake after some tries. The file will remain unchanged.